.env.test

```diff
@@ -1,8 +1,8 @@
 NODE_ENV=test
 PORT=2020
 JWT_SECRET=TEST_SECRET
-MONGODB_URI=mongodb://localhost:27017
-SENDGRID_API_KEYS=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
+MONGODB_URI=mongodb://localhost/FundMylaptop
```

Comment:
I don't think you need to push this file

Reply:
this is .env.test not .env

I pulled it and pushing it back only with little changes. Remember this is a forked repo from HNGi7-Team-Falcon and may have to keep it that way so as not to break code.

Reply:
You're removing the SENDGRID key?

Reply:
The one there before has been blocked for some reason so I replaced with another

Reply:
Note also it is inside` .env.test` not `.env` If anyone needs test the code he will have to copy it to the `.env` to function

Reply:
Actually the the codebase now use the right env file for the right environment.

Reply:
check the package json file and see how `npm start` and `npm test` are implemented

Reply:
I am not understanding you Usman, let us talk on slack to clear things further